backend/app/agents/devrel/github/services/github_mcp_client.py

```
@@ -0,0 +1,105 @@
+                    return {"error": f"MCP server error: {response.status}"}
+
+        except aiohttp.ClientError as e:
+            logger.error(f"Error communicating with MCP server: {e}")
+            return {"error": f"Communication error: {str(e)}"}
+        except Exception as e:
+            logger.error(f"Unexpected error: {e}")
+            return {"error": f"Unexpected error: {str(e)}"}
+
+    async def list_org_repos(self, org: str) -> Dict[str, Any]:
+        if not self.session:
+            raise RuntimeError("Client not initialized. Use async context manager.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)